.changeset/yellow-cats-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+"@tiptap/extension-font-family": patch
```

Comment:
This is also changing the extension-text-style extension right?

Reply:
Oh 😱  I forgot to include it in the changelog! Great catch! Thanks Dom 🙇🏻

Reply:
No worries!